DigitalLearningSolutions.Data/Mappers/MapperHelper.cs

```diff
@@ -6,6 +6,7 @@ public static class MapperHelper
     {
         public static void SetUpFluentMapper()
         {
+            FluentMapper.EntityMaps.Clear();

```

Comment:
Without this, we try and set the entity maps twice with a failure (because we need to start a fake test server). I can't think of any ways this would cause issues to the existing code, but it would be neater if we didn't have to do it. I might look into it later unless anyone has immediate ideas.